### PR TITLE
#5 Adds ability to hire spaces (Hardcoded 'space_id').

### LINKED ISSUE #5: Request to hire a space
```
As a signed-up user
So I can hire a space
I would like to request to hire a space for one night
```

### PR DESCRIPTION
**Pull request by ealitten**
Closes #5

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
```
Comment (ealitten) [inline .idea/misc.xml]: Can we add .idea to .gitignore?

File: strength-bnb.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="RUBY_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (ealitten) [inline strength-bnb.iml]: Also add this to .gitignore

File: app/views/hire.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<form action="/hire" method="post">
+  <input type="text" name="date" placeholder="date">
```
Comment (jenniferemshepherd) [inline app/views/hire.erb:2]: how about a placeholder that is the required date format?
DD/MM/YYYY

Comment (ealitten) [reply app/views/hire.erb:2]: There is no required format, it's just stored as a string in the DB

Comment (jenniferemshepherd) [reply app/views/hire.erb:2]: shall we do it after MVP confirmed?

**Comment by jenniferemshepherd** [comment]
4 x :shipit:

**Pull request merged**